### DISCUSSION TITLE
Depending on network, get net::ERR_CERT_AUTHORITY_INVALID with supabase-js client

### DISCUSSION THREAD

**Discussion by lomaxap** [supabase/supabase#28701] [Questions]
I've seen this error in my application logs since I started using supabase, but have never been able to replicate it. I happened to be working from a public library today and was finally able to produce the error when connected to the library's wi-fi network. When I switch to my phone's hotspot, the error goes away. I'm assuming this isn't a supabase-specific issue, but all other requests my application makes work correctly. Any ideas?

<img width="881" alt="Screenshot 2024-08-16 at 10 44 59 AM" src="https://github.com/user-attachments/assets/1c6edf62-35ce-4293-8ba1-1c1bd862579d">

**Comment by ufukcam** [comment]
Check your server's firewall. Let's Encrypt (for SSL) often tries to renew certificates on port 80. Make sure your server allows connections on port 80 and 443.
